src/bors/command/parser.rs

```diff
@@ -75,13 +77,28 @@ impl CommandParser {
                                         Some(Err(CommandParseError::UnknownCommand(command)))
                                     }
                                     // For `@bors key=<value>`.
-                                    CommandPart::KeyValue { key, .. } => {
+                                    CommandPart::KeyValue { key, value } => {
                                         if key == "r" {
                                             if let Some(result) = parse_approve_on_behalf(&parts) {
                                                 return Some(result);
                                             }
                                         }
 
+                                        if key == "rollup" {
+                                            if ROLLUP_VALUES.contains(&value) {
+                                                return Some(Ok(BorsCommand::Rollup(
+                                                    value.to_string(),
+                                                )));
+                                            } else {
+                                                return Some(Err(
+                                                    CommandParseError::ValidationError(
+                                                        "rollup status can be never/iffy/maybe/always"
```

Comment:
This could be generated from `ROLLUP_VALUES`, to ensure that it doesn't go out of sync.